### ISSUE TITLE
Request for the Pretrained weight files

### ISSUE THREAD

**Issue by aungpaing-gw** [mingyuShin/Depth-metric-learning-3d-object-detection#1]
Hi, Good day to you Sir.

This is really an excellent work and I would like to test out the model.
is it possible to get the pre-trained weight files for this project.

**Comment by mingyuShin** [comment]
I apologize for the late response. I will soon find and upload the pretrained weights.

**Comment by mingyuShin** [comment]
I'm sorry, but I can't find the pretrained weights. However, it is easy to reproduce the results. If you cannot reproduce it, please leave an issue again.
